.eslintrc.js

```diff
@@ -46,18 +46,18 @@ module.exports = {
       files: [
         '.eslintrc.js',
         '.prettierrc.js',
-        'ember-cli-build.js',
-        'index.js',
-        'testem.js',
-        'bin/**',
-        'blueprints/*/index.js',
-        'blueprints/*.js',
-        'config/**/*.js',
-        'lib/**/*.js',
-        'node-tests/**',
-        'tests/dummy/config/**/*.js',
+        'packages/*/ember-cli-build.js',
+        'packages/*/index.js',
+        'packages/*/testem.js',
+        'packages/*/bin/**',
+        'packages/*/blueprints/*/index.js',
+        'packages/*/blueprints/*.js',
+        'packages/*/config/**/*.js',
+        'packages/*/lib/**/*.js',
+        'packages/*/node-tests/**',
+        'packages/*/tests/dummy/config/**/*.js',
       ],
-      excludedFiles: ['addon/**/index.js'],
+      excludedFiles: ['packages/*/addon/**/index.js'],
```

Comment:
Don’t need this any more (because we no longer have `index.js` without slashes or globs in the files array).